Tesla_rm/Tesla_rm/View/SettingsView.swift

```diff
@@ -0,0 +1,152 @@
+// SettingsView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Settings
+struct SettingsView: View {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let backgroundColorName = "Background"
+        static let teslaTextName = "Tesla"
+        static let kmLeftTextName = "187 km"
```

Comment:
```suggestion
        static let teslaTextName = "Tesla"
        static let kmLeftTextName = "187 km"
```
мне чет твои TextName не нравятся вообще, это ж текст, ну накрайняк Title or String